cross/lz4/Makefile

```diff
@@ -0,0 +1,22 @@
+PKG_NAME = lz4
+PKG_VERS = 1.7.5
+PKG_EXT = tar.gz
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
```

Comment:
Please use`PKG_DIST_FILE` for a better tar.gz name. example see: https://github.com/SynoCommunity/spksrc/blob/master/cross/rtmpdump/Makefile#L5
basically something like `$(PKG_NAME )-v$(PKG_VERS).$(PKG_EXT)` should be good ;)

Reply:
Ok, I will try to use PKG_DIST_FILE for lz4 package